package.json

```diff
@@ -38,6 +41,7 @@
     "concurrently": "^8.2.0",
     "eslint-config-curvenote": "^0.0.4",
     "npm-run-all": "^4.1.5",
+    "npm-upgrade-monorepo": "^1.2.6",
```

Comment:
This is nice for upgrading all myst dependencies. Is it too much to have here though? Causes package-lock to grow a lot...